### PR TITLE
[e2e] adding a period to the tested fqdn and improve error readability

### PR DESCRIPTION
**Pull request by AlonaKaplan**
**What this PR does / why we need it**:
The PR contains two commits.
The first one is  adding a period to the tested fqdn.
According to the nslokkup man page - 
```
If the lookup request contains at least one period but doesn't end with a trailing period, append the domain names in the domain search list to the request until an answer is received.
To avoid this, a period is added to our fqdn.
```
    
The second one returns readable `nslookup` error.   

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/vm_startup_test.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ var _ = Describe("Virtual Machines Startup", func() {
 			var nslookupOutput []byte
 			Eventually(func() error {
 				var nslookupErr error
-				nslookupOutput, nslookupErr = exec.Command("nslookup", fmt.Sprintf("-port=%s", dnsPort), fmt.Sprintf("%s.%s.%s.%s", interfaceName, vmiName, testNamespace, domain), dnsIP).CombinedOutput()
+				nslookupOutput, nslookupErr = exec.Command("nslookup", fmt.Sprintf("-port=%s", dnsPort), fmt.Sprintf("%s.%s.%s.%s.", interfaceName, vmiName, testNamespace, domain), dnsIP).CombinedOutput()
```
Comment (oshoval) [inline tests/vm_startup_test.go:91]: Please consider using nosearch instead adding a dot

Comment (AlonaKaplan) [reply tests/vm_startup_test.go:91]: We use dot for the same purpose in the zone file as well. I prefer to stick to this standard.

**Comment by oshoval** [comment]
You might want to add please what happened on CI when there was no dot

**Review by oshoval — APPROVED** [review]
Thanks

**Comment by AlonaKaplan** [comment]
> You might want to add please what happened on CI when there was no dot and no DOMAIN

`** server can't find nic1.vmi-sec-v7tgz.secondary-test-frwp6.vm.kubevirt-prow-jobs.svc.cluster.local: SERVFAIL`
`kubevirt-prow-jobs.svc.cluster.local` domain was appended.

**Pull request merged**